core/src/main/java/org/mule/runtime/core/internal/retry/policies/SimpleRetryPolicy.java

```diff
@@ -170,6 +170,20 @@ public PolicyStatus applyPolicy(Throwable cause) {
     }
   }
 
+  /**
+   * @return how many times a retry should be attempted.
+   */
+  public int getCount() {
+    return count;
+  }
+
+  /**
+   * @return how long to wait between retries.
+   */
+  public Duration getFrequency() {
+    return frequency;
+  }
+
```

Comment:
Let me know if you are ok with exposing these. I figure it is not only harmless but also kind of makes sense in the domain of the class to be able to provide the current values for these parameters. Also this impl class is internal.

Can't just make them package-private because it is in a different package and actually a different module than the test ([ReconnectionTestCase](https://github.com/mulesoft/mule/pull/12465/files#diff-1c456ccef41ce92baac50cdd0b886b5300dc4fd5d32d163b865bb215164e0046)). The test is not unitary but functional.

Ideally we would like to test that these parameters are respected when applying the policy but that would require a lot more complexity by using some fake schedulers, etc. We could do that but I feel like it is outside of this task's scope, and I'm not sure it is completely worth the effort.